docker-compose-celo.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+version: '3'
+services:
+  geth1:
+    image: f2700abb0fa1
+    container_name: celo-geth1
+    ports:
+      - "8545:8546"
```

Comment:
You want to be binding the WS port in the container (see entrypoint.sh) to the `8545` in the host (`8545:8545`). You need to fix this below as well.